README.rst

```diff
@@ -87,6 +90,40 @@ The XML returned by the GSA must contain values for ``portal_name`` and
 ``portal_short_description``, which will be turned into the name and
 description returned by crane's search API.
 
+Solr
```

Comment:
Documentation should indicate the precedence, i.e. if a GSA search is configured it will always take precedence over the Solr serach and we are not providing a union of the results.